docs/product-getting-started-guide/src/main/asciidoc/business-process-proc.adoc

```diff
@@ -11,15 +11,15 @@ The mortgage application scenario is comprised of the following business process
 
 .Prerequisites
 
-Created the *Mortgage_Process* data model. For further information, see <<defining_a_data_model>>.
+Created the *Mortgage Process* data model. For further information, see <<defining_a_data_model>>.
```

Comment:
"You have created the ..." This comes up a few times. Review all prereqs and ensure this consistent passive construction.

 Search and replace "For further" with "For more". Our style says "For more information/details (about)..."